composables/use-book-info.ts

```diff
@@ -49,31 +68,120 @@ export default function ({ nftClassId = '' }: { nftClassId?: string } = {}) {
     }))
   })
 
+  const defaultContentURL = computed(() => {
+    return contentURLs.value.find(url => url.type === 'epub') || contentURLs.value[0]
+  })
+
+  const getReaderRoute = computed(() => ({ nftId, contentURL: inputContentURL }: { nftId?: string, contentURL?: ContentURL }) => {
+    const contentURL = inputContentURL || defaultContentURL.value
+    if (!contentURL) return undefined
+
+    const { url, type, name, index } = contentURL
+    const query: Record<string, string | number> = {
+      nft_class_id: nftClassId,
+      file_url: url,
+      filename: name,
+      index: index,
+    }
+    if (nftId) {
```

Comment:
```suggestion
    if (nftId !== undefined) {
```
Although `'0'` is trueful, but not sure if ppl will accidentally pass a `0` in